modules/encrypt/config/encrypt.php

```diff
@@ -5,14 +5,19 @@
  * string   key     secret passphrase
  */
 
+use Kohana_Encrypt_Engine_Mcrypt as Mcrypt;
+use Kohana_Encrypt_Engine_Openssl as OpenSSL;
+use Kohana_Encrypt_Engine_Sodium as Sodium;
+use Kohana_Encrypt_Engine as Engine;
```

Comment:
Why do you do this? With correct naming this shold not be done. Please change

Reply:
Those are simply aliases for the class names. Because sometimes in the code it's easier to write Engine::CONFIG_KEY rather than Kohana_Encrypt_Engine::CONFIG_KEY, especially when you type it multiple times

Reply:
Yes i know,but i don't get why doing it in Configuration file, did that no where else

Reply:
Ok, that's not a problem. I can fix this.